### PR TITLE
Fixed bugs when creating new registry keys

### PR DESCRIPTION
**Pull request by SuperMarioo**
Fixed Errors when creating new Registry keys , and changed erroraction
to SilentlyContinue as it was reporting errors if registry was not found
.

---

This change is [Reviewable](https://reviewable.io/reviews/powershell/xsqlps/9)

### CODE REVIEW

File: DSCResources/MSFT_xSqlAlias/MSFT_xSqlAlias.psm1 [hunk 1]
```diff
@@ -27,7 +27,7 @@ function Get-TargetResource
 
     if ($null -ne (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\MSSQLServer\Client\ConnectTo' -Name "$SQLServerName" -ErrorAction SilentlyContinue))
     {
-        $ItemValue = Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\MSSQLServer\Client\ConnectTo' -Name "$SQLServerName";
+        $ItemValue = Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\MSSQLServer\Client\ConnectTo' -Name "$SQLServerName" -ErrorAction SilentlyContinue;
```
Comment (KarolKaczmarek) [inline DSCResources/MSFT_xSqlAlias/MSFT_xSqlAlias.psm1:30]: in what scenario was the error returned here? Wasn't it being filtered by the if statement on line 28?

**Comment by TravisEz13** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xsqlps/9)_

**Comment by narrieta** [comment]
@SuperMarioo : Is this PR still active or can we close it?

**Comment by kwirkykat** [comment]
@SuperMarioo Closing all PRs as this module has been deprecated since it's functionality is covered by [xSqlServer](https://github.com/PowerShell/xSqlServer)

**Pull request closed without merging**